Add format and style changes to current article content (#31)
nipy/heudiconv-joss-paper: paper.md

# Overview of HeuDiConv functionality

HeuDiConv was initially developed to implement logic commonly used across labs (grouping DICOMs, extracting metadata, converting individual sequences, populating standard BIDS files, etc.) while allowing individual groups to customize **how** files should be organized and named while driving custom decisions through the conventions and desires of those individual groups.
Such decision making is implemented in *HeuDiConv heuristics*, which are implemented as Python modules following some minimalistic specified interfaces documented in HeuDiConv documentation (https://heudiconv.readthedocs.io/en/latest/heuristics.html).

> HeuDiConv, if instructed to operate in BIDS mode (`--bids` flag) with a heuristic providing base naming instructions, organizes the files in the hierarchy defined by the BIDS standard. It also ensures files are named according to the BIDS specifications, including complex split files such as those associated with multi-echo sequences.

Reviewer: (Un)fortunately HeuDiConv doesn't really place files into hierarchy -- it remains for a heuristic to do that but indeed HeuDiConv provides some placeholders ... which we did not document much, so submitted https://github.com/nipy/heudiconv/pull/681  . So, what about

```suggestion
HeuDiConv, if instructed to operate in BIDS mode (`--bids` flag) with a heuristic providing base naming instructions, and helpers to organize the files in the hierarchy defined by the BIDS standard.
It also ensures files are named according to the BIDS specifications, including complex composite recordings such as those associated with multi-echo sequences.
```

# Adoption and usage

> As citeable resource [RRID:SCR_017427](https://scicrunch.org/resolver/RRID:SCR_017427), Heudiconv has already [6 mentions in papers](https://scicrunch.org/resolver/SCR_017427/mentions?q=&i=rrid:scr_017427) at time of writing.

Reviewer: I feel like it needs one but I could be wrong:

```suggestion
As a citeable resource [RRID:SCR_017427](https://scicrunch.org/resolver/RRID:SCR_017427), Heudiconv has already [6 mentions in papers](https://scicrunch.org/resolver/SCR_017427/mentions?q=&i=rrid:scr_017427) at time of writing.
```

There is a growing number of downloads from PyPI and uses of HeuDiConv (see \autoref{fig:usage}).

> Over 40 BIDS datasets were converted over to BIDS with HeuDiConv at Dartmouth Brain Imaging Center (DBIC), using the ReproIn heuristic currently developed there.

Reviewer: ```suggestion
Over 40 BIDS datasets were converted over to BIDS with HeuDiConv at Dartmouth Brain Imaging Center (DBIC), using the ReproIn heuristic developed there.
```
I think it would be easier to swallow this way
